tests/c/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 cmake_minimum_required(VERSION 3.5)
 
 find_package(GTest)
```

Comment:
How do we prefer CONFIG if it's available over MODULE here?

Reply:
Ok I remember the flag you set elsewhere. We should keep all those similar changes together. Otherwise it's hard to see what's going on